ingestc.js

```diff
@@ -0,0 +1,105 @@
+/* -----------------------------------------------------------------------------
+ * @copyright (C) 2020, Alert Logic, Inc
+ * @doc
+ *
+ * HTTP client for Ingest service.
+ *
+ * @end
+ * -----------------------------------------------------------------------------
+ */
+'use strict';
+
+const AlServiceC = require('./al_servicec').AlServiceC;
+
+/**
+ * @class
+ * HTTPS client for Alert Logic Ingest service.
+ *
+ * @constructor
+ * @param {string} apiEndpoint - Alert Logic API hostname.
+ * @param {Object} aimsCreds - Alert Logic API credentials object, refer to AimsC.
+ * @param {string} [aimsCreds.access_key_id] - Alert Logic API access key id.
+ * @param {string} [aimsCreds.secret_key] - Alert Logic API secret key.
+ *
+ */
+class IngestC extends AlServiceC {
+    constructor(apiEndpoint, aimsCreds, functionType, retryOptions) {
+        super(apiEndpoint, 'ingest', 'v1', aimsCreds, retryOptions);
+        this._functionType = functionType ? functionType : 'lambda_function';
+    }
+
+    sendSecmsgs(data) {
+        let payload = {
+            json : false,
+            headers : {
+                'Content-Type': 'alertlogic.com/cwe-json',
+                'x-invoked-by' : this._functionType,
+                'Content-Encoding' : 'deflate',
+                'Content-Length' : Buffer.byteLength(data)
+            },
+            body : data
+        };
+        return this.post(`/data/secmsgs`, payload);
+    }
+
+    sendVpcFlow(data) {
```

Comment:
As far as I know, this message type doesn't exist? Should we remove it? If we do VPC flow logs in the future, won't they be `logmsgs`?

Reply:
it still exists. We may want to reuse it.

Reply:
cool. Lets keep it then